### DISCUSSION TITLE
Can i handle POST request in page.tsx file?

### DISCUSSION THREAD

**Discussion by undefine-man** [vercel/next.js#67128] [Help]
### Summary

From Doc said , _A page is UI that is unique to a route_.
Can i handle POST request in page.tsx file?
I want to use the same url path the handle UI response  also POST request, but i test below, it doesn't for the POST function.
It is possible to handle api route and ui page response in page.tsx file ?
![image](https://github.com/vercel/next.js/assets/24623544/0e343361-b8c2-4f0d-87d1-62e5dd2b9ad6)

**Comment by versecafe** [comment]
@undefine-man You can't, you also can't have a `route.ts` and `page.tsx` file at the same routing position. If you want to handle server side actions triggered by the client you can always use a server action instead.
